index.html

```diff
@@ -89,15 +89,15 @@ <h1 class="text-white font-rajdhani text-left text-xl md:text-3xl">
                             1. Who is Darkbeast?
                         </h1>
                         <p>
-                            He is commonly referred to by various aliases such as: shadowbeast, nightshade, darknight, shadow skid, among others. Despite being a 15 year old kid, he portrays himself as an adult in his early twenties. However, his real name is Faiz Patel and he lives in Kolkata, India.
+                            He is commonly referred to by various aliases such as: BeastGamer81, darknight, darknight81, among others. Despite being a 15 year old kid, he portrays himself as an adult in his early twenties. However, his real name is Faiz Patel and he lives in Maharashtra, India.
                         </p>
                     </div>
                     <div class="flex flex-col">
                         <h1 class="text-white font-rajdhani text-left text-xl md:text-3xl">
                            2. What has he done?
                         </h1>
                         <p>
-                            His criminal activities encompass a wide array of offenses, including exploiting trial periods without authorization, fabricating counterfeit credit cards, manipulating server providers, circumventing FOSS License regulations, and numerous others.

```

Comment:
dlgtm

Reply:
I never fabricated counterfeit credit cards and trials

Reply:
however if you want to revert it, you can.